src/clients/components/GoogleMaps.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+
+import { useState, useEffect } from "react";
+import { useMantineColorScheme } from "@mantine/core";
+import { mapTheme, loader } from "~/utils";
+
+interface GoogleMapProps {
+  address: string;
+}
+
+export const GoogleMaps = ({ address }: GoogleMapProps) => {
+  const [map, setMap] = useState<google.maps.Map>();
+  const { colorScheme } = useMantineColorScheme();
+
+  useEffect(() => {
+    const fetchMap = async () => {
+      await loader.load().then(async () => {
+        const geocoder = new window.google.maps.Geocoder();
+        await geocoder.geocode({ address }, (results, status) => {
+          if (status === "OK" && results !== null) {
+            const mapOptions = {
+              center: results[0]?.geometry.location,
+              zoom: 16,
+              styles: colorScheme == "dark" ? mapTheme.dark : mapTheme.light,
+            };
```

Comment:
@devjoylee I could barely remember but I think we can add the zoom feature in this GoogleMap by changing `zoom` value in `const mapOptions` (The value of `zoom` in `const mapOptions` is now 16 (constant value).

I've recognized that we can zoom in or out map by clicking `+` or `-` button in the map after viewing it, so this is not 100% necessary as well.